app/test_main.py

```diff
@@ -1 +1,46 @@
-# write your code here
+from unittest import mock
+from typing import Callable
+
+import pytest
+
+from app.main import can_access_google_page
+
+
+class TestCanAccessGooglePage:
+    @pytest.mark.parametrize(
+        "mocked_connection, mocked_url, result, url",
+        [
+            (True, True, "Accessible",
+             "https://mate.academy/courses/python"),
+
+            (True, False, "Not accessible",
+             "https://mate.academy/courses/python"),
+
+            (False, True, "Not accessible",
+             "https://mate.academy/courses/python"),
+
+            (False, False, "Not accessible",
+             "https://mate.academy/courses/python"),
```

Comment:
i mean coudl u pass this url directly to the func in line 46
cause u are using the same url every time